lib/python/rose/apps/rose_bunch.py

```diff
@@ -265,13 +288,17 @@ def run(self, app_runner, conf_tree, opts, args, uuid, work_files):
 
         if abort and commands:
             for key in self.invocation_names:
+                notrun += 1
                 cmd = commands.pop(key).get_command()
                 app_runner.handle_event(NotRunEvent(key, cmd),
                                         prefix=self.PREFIX_NOTRUN)
 
         if self.dao:
             self.dao.close()
 
+        # Report summary data in job.out file
+        app_runner.handle_event(SummaryEvent(run_ok,run_fail,run_skip,notrun))
```

Comment:
Yeah, my bad. Ran tests and then renamed some variables to be clearer after that.